docs/product-installation-guide/src/main/asciidoc/openshift-bxms-con.adoc

```diff
@@ -6,8 +6,8 @@ With {PRODUCT} {RELEASE}, the {PRODUCT} on OpenShift files are provided in the `
 
 * The `application-template` directory contains OpenShift templates.
 * The `BPMS-7.0.0.Beta02` directory contains productized binaries which you can use to build your own images.
-* The `openshift-image-source` directory contains scripts that can be used to create OpenShift images based on previous Docker images.
-* The `standalone-image-source` directory contains scripts that can be used to create Docker images without any OpenShift dependencies.
+// *  The `openshift-image-source` directory contains scripts that can be used to create OpenShift images based on previous Docker images.
```

Comment:
The openshift-image-source and standalone-image-source will stay in openshift zip. The only thing dropped are product image TAR files.